helper/krylovHelper.h

```diff
@@ -0,0 +1,129 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+#include <iomanip>
+
+#include "krylovTimeEvolver.h"
+
+#ifdef USE_HDF
+#include <H5Cpp.h>
+#include "krylovHelper.h"
+using namespace H5;
+#endif
+
+
+//parent class
+class parameter
```

Comment:
It is my understanding that usually each class should be in a separate file. But this would lead to quite a number of files, so I'm not insisting...

Reply:
I think Java enforces something like that. I would keep it in the same file as long as it only defines helpful new datatypes (with almost no class functions) and it belongs to the same functional category.